specs/_features/eip7594/p2p-interface.md

```diff
@@ -322,6 +322,6 @@ Requests the MetaData of a peer, using the new `MetaData` definition given above
 
 A new field is added to the ENR under the key `csc` to facilitate custody data column discovery.
 
-| Key    | Value                               |
-|--------|-------------------------------------|
-| `csc`  | Custody subnet count, uint8 integer |
+| Key    | Value                                    |
+|--------|------------------------------------------|
+| `csc`  | Custody subnet count, big endian integer |
```

Comment:
May be we could be more explicit about type (maximum value) and encoding here? 
Even if we refer to the [ENR spec](https://github.com/ethereum/devp2p/blob/master/enr.md) it looks not that explicit about encoding of `big endian integer` as well: is the `0x0040` or `0x00000040` right encoding of `uint(64)`?